css/style.css

```diff
@@ -131,8 +260,8 @@ body {
 }
 
 .search_bar input[type=submit] {
-	background: #0f0;
-	padding: 15px 30px;
+	background: #92ed39;
```

Comment:
I changed the colour to be a bit less bright of a green to be easier on the eyes, I also changed the padding so that it was the right size next to the search bar